cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetDTO.java

```diff
@@ -74,6 +74,9 @@
 
     private BudgetVMConfigurationDTO vmConfiguration;
 
+    @ApiModelProperty( value = "Whether the budget is for a delayed subscription downgrade", readOnly = true )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "readOnly"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)